includes/wp-admin/class-settings.php

```diff
@@ -358,7 +358,7 @@ public static function settings_page() {
 				\wp_enqueue_media();
 				\wp_enqueue_script( 'activitypub-header-image' );
 				break;
-			case 'settings':
+			default:
```

Comment:
If we run it on default, we should probably run it much earlier. Can you help me understand why you don't want to scope it to the settings tab?

Reply:
because we call the getting started on the welcome page.

Reply:
Ah! Part of #1666 was the proposal to have help links link to the settings tab, resulting in a new pageload and avoiding the flash of an updated checklist. So they should always end up here

Reply:
but it does not work like that

Reply:
It looks like [the change that added the tab parameter to the getting-started link](https://github.com/Automattic/wordpress-activitypub/pull/1666/commits/f134a9fd3a6a7df9dab1b6013105a47311aed6ab) was [reverted somehow in a merge commit](https://github.com/Automattic/wordpress-activitypub/pull/1666/commits/c85fb7d5205af67f193aaf3be33df9ca08ab4308) :/

Reply:
If we keep them on the Welcome page we need some sort of JS to update the option and make all the UI changes that come with it, otherwise it will load the updated page, with the step marked as complete, before the help tab slides down.
By linking to the settings tab, we avoid the flash of updated checklist, but still get a page refresh

Reply:
with my change not... you simply redirect to the welcome page with the help-tab param... so technically it is a redirect, but to the same page..

Reply:
Interesting. So it saves the option after rendering the checklist?
It's still a bit funky, with it being not checked off after opening the help tab.

https://github.com/user-attachments/assets/051f185d-c2c3-43c3-913d-06146bfcb417

Reply:
The checklist item not updating after clicking the button feels pretty unpolished, no?

Reply:
true... but can we make it happen? 👀